src/Microsoft.AspNetCore.All/build/aspnetcore-store-2.0.0-linux-x64.xml

```diff
@@ -0,0 +1,171 @@
+<StoreArtifacts>
+   <Package Id="libuv"  Version ="1.10.0"/>
```

Comment:
Nit: is the extra space intentional? `Version=` vs `Version =`

Reply:
Never paid attention to the output of the dotnet-store manifests. This is a dotnet-store bug in their format.